NuWeather/config.py

```diff
@@ -60,15 +60,24 @@ class NuWeatherTemperatureDisplayMode(registry.OnlySomeStrings):
         F/C means show "50F/10C", C means display only Celsius, and so on.""")))
 
 BACKENDS = ('darksky', 'apixu')
+GEOBACKS = ('nominatim', 'google')
```

Comment:
I would prefer `GEOCODE_BACKENDS` instead of an arbitrary abbreviation

Also, maybe `googlemaps` would be a more descriptive keyword for that API?